src/views/AdditionalInformation/RelatedRequestsTable.tsx

```diff
@@ -40,23 +34,20 @@ import user from 'utils/user';
 import { NotFoundPartial } from 'views/NotFound';
 
 import { LinkedRequestForTable } from './linkedRequestForTable';
+import useRelatedRequests from './useRelatedRequests';
 
 const RelatedRequestsTable = ({
-  systemIntakeID,
+  requestID,
+  type,
   pageSize = 10
 }: {
-  systemIntakeID: string;
+  requestID: string;
+  type: 'trb' | 'itgov';
```

Comment:
Can use [RequestType](https://github.com/CMS-Enterprise/easi-app/blob/main/src/types/requestType.ts#L2)

Reply:
oh nice! thank you